lib/index.ts

```diff
@@ -274,7 +280,10 @@ async function getManifestFiles(
     .filter((i) => i.contents !== "");
 }
 
-function parseAnalysisResults(targetImage, analysis) {
+function parseAnalysisResults(
+  targetImage,
```

Comment:
what is the type of `targetImage`? is it string?

Reply:
I don't know, it's untyped